core/templates/app.constants.ts

```diff
@@ -163,7 +163,9 @@ export const AppConstants = {
 
   SITE_NAME: 'Oppia.org',
 
-  DEFAULT_PROFILE_IMAGE_PATH: '/avatar/user_blue_72px.webp',
+  DEFAULT_PROFILE_IMAGE_WEBP_PATH: '/avatar/user_blue_72px.webp',
+
+  DEFAULT_PROFILE_IMAGE_PNG_PATH: '/avatar/user_blue_72px.png',
```

Comment:
Should we modify these images to have the same dimensions as regular images?

Reply:
I have updated this part.